docs/deploy/providers/stormkit.md

```diff
@@ -6,6 +6,11 @@
 Integration with this provider is possible with zero configuration. ([Learn More](/deploy/#zero-config-providers))
 :::
 
-::: warning Work in Progress
-Documentation for this section is not yet complete.
-:::
+Nitro supports [Stormkit](https://www.stormkit.io/) out of the box.
```

Comment:
There is already an alert above isn't this duplicate? I'm fine to reword alert if you like BTW :)

Reply:
The alert looks nice. The problem with zero config is there is so little to document 😆